### PR TITLE
Add test runtime project.json support running against desktop

### PR DESCRIPTION
**Pull request by tarekgh**

### CODE REVIEW

**Comment by mmitche** [comment]
@dotnet-bot test this please

**Comment by tarekgh** [comment]
test Innerloop Windows_NT Release Build and Test please

**Comment by tarekgh** [comment]
@weshaggard could you please have a look? this make a separate project.json for desktop build as having separate run times in one project.json not working for Linux builds.

File: dir.props [hunk 1]
```diff
@@ -85,6 +85,8 @@
 
   <!-- Test runtime -->
   <PropertyGroup>
+    <TestRuntimeProjectJson Condition="'$(TestRuntimeProjectJson)' == '' And '$(TargetGroup)' == 'net46'">$(SourceDir)Common/test-runtime/desktop/project.json</TestRuntimeProjectJson>
```
Comment (weshaggard) [inline dir.props]: Can you please follow the naming convention and name the directory net46 instead of desktop.

Comment (tarekgh) [inline dir.props]: I'll fix that.

**Pull request merged**